docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,20 @@
 
 TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
 
+== OpenShift 4.11
+
+More flexible alert routing for User Workload Monitoring::
+
+You can now https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring[enable alert routing] as well as spin up a https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts[dedicated Alertmanager] for User Workload Monitoring. Further retention can now be defined based on https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-confgure-retention-size-for-metrics-storage[disk usage].
```

Comment:
```suggestion
You can now https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring[enable alert routing] as well as spin up a https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts[dedicated Alertmanager] for User Workload Monitoring.

Additionally, metrics retention can now be defined based on https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-confgure-retention-size-for-metrics-storage[disk usage].
```
Technically the note about metrics retention based on size should probably be its own entry, as it doesn't relate to alert routing strictly speaking.